From cba2bc2f7ebc72754c2da81c86c7cb3d47944360 Mon Sep 17 00:00:00 2001
From: "Wang, Apple" <apple.wang@intel.com>
Date: Wed, 31 Dec 2025 01:40:58 +0000
Subject: [PATCH 5/6] Add NVL platform support to build configuration

Added 'NVL' to the list of supported graphics platforms in the build system configuration.

OSV: https://jira.devtools.intel.com/browse/PKT-20292
Upstream-status: Backport
Link: https://github.com/intel/vpl-gpu-rt/commit/3a2ec9d00169fde835cabf6dcdac4bcdf91f25bf
---
 builder/BuildOptionsPlatform.cmake | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/builder/BuildOptionsPlatform.cmake b/builder/BuildOptionsPlatform.cmake
index 774541ea..8a636ec6 100644
--- a/builder/BuildOptionsPlatform.cmake
+++ b/builder/BuildOptionsPlatform.cmake
@@ -1,4 +1,4 @@
-# Copyright (c) 2024 Intel Corporation
+# Copyright (c) 2024-2025 Intel Corporation
 #
 # Permission is hereby granted, free of charge, to any person obtaining a copy
 # of this software and associated documentation files (the "Software"), to deal
@@ -24,7 +24,7 @@ message( STATUS "Configuration of Supported Platforms" )
   # Supported Platforms
   set(BS_GFX_PLATFORMS_SUPPORTED
       "TGL" "RKL" "DG1" "ADL" "RPL" "DG2" "MTL" "ARL"
-      "LNL" "BMG" "PTL"
+      "LNL" "BMG" "PTL" "NVL"
   )
 
 message( STATUS "Supported platforms = ${BS_GFX_PLATFORMS_SUPPORTED}" )
-- 
2.43.0

